### PR TITLE
Skip actor pre_check if actor not present

### PR DESCRIPTION
**Pull request by lbarcziova**
This should make the behaviour the same as it was previously (before handler pre_checks refactoring
https://github.com/packit/packit-service/blob/da94cc9b511843d73e9b94d20677f0b1d1426bf8/packit_service/worker/jobs.py#L422): if the actor is None, the actor pre_check will be skipped (e.g. pushes to branch, releases).

---

RELEASE NOTES BEGIN

RELEASE NOTES END

### CODE REVIEW

**Comment by mfocko** [comment]
But someone »does« the push or release, don't they?

**Comment by lbarcziova** [comment]
If I recall correctly, we previously did the check only for PRs since if a person commits or releases, he/she has also write access to the repo.

**Review by mfocko — APPROVED** [review]

File: packit_service/worker/checker/abstract.py
Comment (lachmanfrantisek) [inline packit_service/worker/checker/abstract.py:43]: Aren't we checking also other non-actor conditions? (I went through the code really quickly and am not sure about the `pre_check` / `_pre_check` call chain...)

**Review by lachmanfrantisek — APPROVED** [review]
Thanks!

File: packit_service/worker/checker/abstract.py
Comment (lachmanfrantisek) [reply packit_service/worker/checker/abstract.py:43]: e.g. project allowed for internal TF,..

Comment (lbarcziova) [reply packit_service/worker/checker/abstract.py:43]: I think only `_pre_check`s in subclasses [here](https://github.com/packit/packit-service/blob/d126fe4839f54c7b4ab7e6bd4232d2e709930208/packit_service/worker/checker/copr.py#L90) or [here](https://github.com/packit/packit-service/blob/d126fe4839f54c7b4ab7e6bd4232d2e709930208/packit_service/worker/checker/testing_farm.py#L63) are called, but I would wait for Maja's approval since she knows the code the best

Comment (majamassarini) [reply packit_service/worker/checker/abstract.py:43]: Thanks Laura for fixing this. 
I did not realize I had changed the actor checker behaviour. You are right, before my changes it would have returned `False` only with a valid `actor` field.

Before my changes this code were called for every handler, but most of them just returned `True`, now it is called just for this two handlers Laura linked here.

**Review by majamassarini — APPROVED** [review]

**Pull request merged**
